src/actfountain.cpp

```diff
@@ -215,6 +215,7 @@ void actFountain(Entity* my)
 					}
 					messagePlayer(i, language[474]);
 					my->skill[0] = 0; //Dry up fountain.
+                    serverUpdateEntitySkill(my, my->skill[0]);
```

Comment:
Out of curiosity, is there a `Sint32&` defined for this in class Entity in Entity.hpp?
E.g. `Sint32& fountainStatus`?

Reply:
Also, it looks like the formatting for this line still needs to be fixed?